ports/opentyrian2000/port.json

```diff
@@ -0,0 +1,29 @@
+{
+    "version": 2,
+    "name": "opentyrian2000.zip",
+    "items": [
+        "Open Tyrian 2000.sh",
+        "opentyrian2000"
+    ],
+    "items_opt": null,
+    "attr": {
+        "title": "Open Tyrian 2000",
+        "porter": [
+            "t0b10"
+        ],
+        "desc": "OpenTyrian2000 is an open-source port of the DOS game Tyrian. It is a fork of OpenTyrian, with the end goal being to replicate the experience of Tyrian 2000 as closely as possible.\n\nTyrian is an arcade-style vertical scrolling shooter.  The story is set in 20,031 where you play as Trent Hawkins, a skilled fighter-pilot employed to fight MicroSol and save the galaxy.\n\nTyrian features a story mode, one- and two-player arcade modes, and networked multiplayer.",
+        "inst": "OpenTyrian2000 requires the Tyrian 2000 data files, which have been released as freeware: https://www.camanis.net/tyrian/tyrian2000.zip\n\nAlternatively, the game is also available free of charge on GOG: https://www.gog.com/en/game/tyrian_2000\n\nGame data must be placed in the 'tyrian2000' subdirectory.",
+        "genres": [
+            "action",
+            "arcade"
+        ],
+        "image": "screenshot.jpg",
```

Comment:
This field should not be filled out.
If you use the json generator and fill it with dummy data you'll see what field need to be filled.
Or better use the generator for the json =)
https://portmaster.games/port-json.html